### DISCUSSION TITLE
error in compilation of rheotool

### DISCUSSION THREAD

**Discussion by rgovindm** [fppimenta/rheoTool#117] [General]
I am getting this error in compilation of rheotool of90 and of70 also. (both installed on respective versions of OpenFoam9 and Openfoam7  in Ubuntu 20.04.6LTS). 
Please help me in sorting this error out.

![image](https://github.com/fppimenta/rheoTool/assets/89021781/6911ba29-5126-4354-a64f-75ff5f663449)

**Comment by fppimenta** [comment]
The error tells you that PETSC (consequently Hypre) is not installed on your system. You either missed that step or the installation ended in error.

NOTE AND GENERAL ADVICE: today, 26/11/2023, I installed ubuntu 22.04.06 LTS + openfoam 9.0 from scratch, following standard instructions, and could confirm that **there is no compilation error** of rheoTool over the whole process. Anyone following the same procedure (as in the user-guide) should get a successful compilation. Installing things (OS+openfoam+rheoTool & dependencies) from scratch on a VBox is a simple way to see that the instructions are working and to possibly help debugging an unsuccessful installation on pre-existing systems.

**Reply by rgovindm** [reply]
Thank you sir for your reply. Error is sorted now. It's working now.
